### PR TITLE
Add test for diameter maximum number of diam req

### PR DESCRIPTION
**Pull request by TeoV**

### CODE REVIEW

File: data/conf/samples/diamagentmaxconn/cgrates.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
+{
```
Comment (danbogos) [inline data/conf/samples/diamagentmaxconn/cgrates.json]: Please rename the file to diamactreqs

**Comment by danbogos** [comment]
Thanks!
